frontend/legacyPartCollectionRoutes.js

```diff
@@ -14,12 +14,16 @@ function mapPartItemTypes() {
    const legacyTagsWithNoItemType = legacyFilterSlugs['tagsWithNoItemType'];
    // Redirect tags we don't support anymore to the base product list page.
    const legacyToParent = legacyTagsWithNoItemType.map((oldTag) => ({
-      source: `/Parts/:slug/${oldTag}`,
+      source: `/Parts/:slug/${escape(oldTag)}`,
       destination: `/Parts/:slug`,
       permanent: true,
    }));
 
    return [...oldToNew, ...legacyToParent];
 }
 
+function escape(str) {
```

Comment:
Do we have a convenient place (utils package?) to move this function so it can be shared with the other usage below? This will also end up giving it a more descriptive name `escapeStringForRegex()` or something.

Reply:
We don't have an existing place, but we can make a new place.